client_app/src/components/shelter/ShiftsModal.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'; 
+import Modal from 'react-modal';
+import { shiftListed } from './shiftsListed.tsx';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEnvelope, faCircleXmark } from '@fortawesome/free-solid-svg-icons';
+import "../../styles/shelter/ShiftsModal.css"
+
+export const ShiftsModal = props => {
```

Comment:
These are because it isn't integrated yet with the backend, and the props value and the setVolunteerList will be needed when that is merged with Nhi's code. And the 3rd one is about the actions package we use for linting uses a depreciated nodejs version.